### DISCUSSION TITLE
PLOT3D output file from FDS

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13804] [Q&A]
I tried to visualize the FDS output plot3D files  in Paraview . The 5 variables written by FDS ( velocities (vx, vy, vz), Temperature and species concentrations ) mismatches the plot3D  output variables.  The variables in Plot3D for "momentum" matches the velocity , But the gas concentrations and Temperatures  are missing in the field. Anyone had a fix to it .

**Comment by johodges** [comment]
There was some discussion on this earlier in the year. The data will be what you told FDS to output but Paraview assumes that the data fields are the ones from the original NASA specification.

              I don't think we have control over this. The names that Paraview lists are the names of the quantities that Plot3D was originally intended to carry. Plot3D is a format developed by NASA to hold generic CFD data.

_Originally posted by @mcgratta in https://github.com/firemodels/fds/issues/12668#issuecomment-2015674858_
